clar2wasm/src/datastore.rs

```diff
@@ -255,11 +263,17 @@ impl ClarityBackingStore for Datastore {
         "".to_string()
     }
 
-    fn insert_metadata(&mut self, contract: &QualifiedContractIdentifier, key: &str, value: &str) {
+    fn insert_metadata(
+        &mut self,
+        contract: &QualifiedContractIdentifier,
+        key: &str,
+        value: &str,
+    ) -> Result<()> {
         // let bhh = self.get_open_chain_tip();
```

Comment:
It's not part of the pr, but deleting the commented-out code would be better.

Reply:
Deleted in 2687ef9